integration_tests/test_store_release.py

```diff
@@ -105,13 +113,20 @@ def test_release_with_login_multiarch(self):
         expected = r'.*amd64   0.1      16        stable     -          -.*'
         self.assertThat(output, MatchesRegex(expected, flags=re.DOTALL))
 
+    @unittest.skipUnless(
+        (os.getenv('TEST_SNAP_WITH_TRACKS', 'fake') ==
```

Comment:
Please verify the integration test using the staging store and using the fake server